Cargo.toml

```diff
@@ -19,6 +19,8 @@ futures = { version = "0.3.9", optional = true }
 async-stream = { version = "0.3.0", optional = true }
 thiserror = "1.0.23"
 serde = { version = "1.0", features = ["derive"] }
+log = "0.4.14"
+env_logger = "0.9.0"
```

Comment:
We need `env_logger` only for tests, maybe move to `[dev-dependencies]`?

Reply:
Right!